### PR TITLE
Foundations, Installations, Prerequisites: Modify link [usage share of web browsers]

### PR DESCRIPTION
**Pull request by blltt**
gs.statcounter.com does not support https.  
Navigating to the URL https://gs.statcounter.com/browser-market-share#monthly-202110-202110-bar results in:
This site can’t provide a secure connection
gs.statcounter.com sent an invalid response.
ERR_SSL_PROTOCOL_ERROR
Furthermore, statcounter.com is included in the D_Me_Malv feed, group DNSBL_Malicious (pfBlockerNG DNSBL).  
Whether it belongs there or not I don't know, but I won't whitelist without https.
I'm recommending a wikipedia page that I believe contains what you want.

Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [ ] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [ ] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX

**2. This PR:**

**3. Additional Information:**

### CODE REVIEW

File: foundations/installations/prerequisites.md [hunk 1]
```diff
@@ -257,7 +257,7 @@ Currently there is a bug preventing CloudReady v83.4 from successfully installin
 #### Why Google Chrome?
 
 Because our lessons use Google Chrome, and Chrome/Chromium are overwhelmingly used by developers and consumers, the recommendations we make are very intentional.
-Look at this [breakdown of browser market shares](https://gs.statcounter.com/browser-market-share#monthly-202110-202110-bar) and see what other folks are using the most
+Look at this [usage share of web browsers](https://en.wikipedia.org/wiki/Usage_share_of_web_browsers) and see what other folks are using the most
```
Comment (thatblindgeye) [inline foundations/installations/prerequisites.md:260]: I think we should link to the exact section where the usage share is listed (the Summary Tables section). Additionally, would you mind adding a period at the end of this sentence?

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
I'd agree with this change, more because the previous link (even if technically accurate) was showing data from roughly 6 months ago, whereas this Wikipedia article I imagine will always be more up to date.

I only had one change request below.

File: foundations/installations/prerequisites.md (hunk 1 above)
Comment (blltt) [reply foundations/installations/prerequisites.md]: Done.  New [commit](https://github.com/TheOdinProject/curriculum/pull/23897/commits/6619bfdd9f7e1297583aa6450ce31e15d785931d)

**Review by thatblindgeye — APPROVED** [review]

**Comment by thatblindgeye** [comment]
Thanks for the changes! Just an FYI in the future, please be sure the PR template is filled out correctly (e.g., the reason for the PR is listed in the Because step, and what the PR does is listed in the second step).

**Pull request merged**